.circleci/config.yml

```diff
@@ -146,7 +146,7 @@ jobs:
             . venv/bin/activate
             pip install --upgrade pip
             pip install grpcio-tools==1.13.0  --progress-bar=off
-            pip install mypy-protobuf --progress-bar=off
+            pip install mypy-protobuf==1.16.0 --progress-bar=off
```

Comment:
This was causing discrepancies between locally generated files and the CI ones, due to https://github.com/dropbox/mypy-protobuf/pull/92. I can undo this and apply it in another PR to cutdown on noise (it touches all the .pyi files) if that's better.

Reply:
Yeah, maybe merge this protobuf change before the sensor change.

Reply:
https://github.com/Unity-Technologies/ml-agents/pull/2757